ngsderive/commands/strandedness.py

```diff
@@ -193,35 +175,40 @@ def determine_strandedness(
                 state_logs = ";".join([f"{state}={n}" for state, n in states.items()])
                 rg_log += f"{rg}:{state_logs} "
 
-            logging.debug("    - {}".format(rg_log))
+            logger.debug(f"    - {rg_log}")
 
-            for rg in this_genes_evidence.keys():
-                for state in this_genes_evidence[rg].keys():
-                    overall_evidence[rg][state] += this_genes_evidence[rg][state]
+            for rg, evidence in this_genes_evidence.items():
+                for state, n in evidence.items():
+                    overall_evidence[rg][state] += n
 
             n_tested_genes += 1
             n_reads_observed += reads_in_gene
         else:
-            logging.debug(
-                "    - Read count too low ({} < {})".format(
-                    reads_in_gene, minimum_reads_per_gene
-                )
+            logger.debug(
+                f"    - Read count too low ({reads_in_gene} < {minimum_reads_per_gene})"
             )
 
+    rgs_in_header_not_in_seq = validate_read_group_info(
+        {rg for rg in overall_evidence if rg not in {"overall", "unknown_read_group"}},
```

Comment:
Same thing here. In fact, since these are used across multiple files, they may need to be in some sort of constants file.